test/test_transforms_tensor.py

```diff
@@ -119,8 +118,12 @@ def test_random(func, method, device, channels, fn_kwargs, match_kwargs):
 
 
 @pytest.mark.parametrize("seed", range(10))
-@pytest.mark.parametrize("device", cpu_and_gpu())
-@pytest.mark.parametrize("channels", [1, 3])
+# Temporary disable CUDA device
+# https://github.com/pytorch/vision/issues/5873
```

Comment:
I'm not sure temporarily disabling the CUDA execution is the way to go here. If there is no other way we can fix it on our side, we should consider reverting https://github.com/pytorch/pytorch/pull/73433 which broke TorchVision.

cc @albanD @jbschlosser

Reply:
Just few more info on failures. Failures with ColorJitter are flaky are failing on batched tensor tests: https://github.com/pytorch/vision/blob/6d85d74be6ac72b2ac3057d85d8ae0004fa0de3f/test/test_transforms_tensor.py#L86
https://github.com/pytorch/vision/blob/6d85d74be6ac72b2ac3057d85d8ae0004fa0de3f/test/test_transforms_tensor.py#L46

Failures are flaky as tests are failing when run all of them and pass if run one by one...